packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts

```diff
@@ -387,23 +387,26 @@ export class StatusBar
 
     private onLayoutComplete(opts: _ModuleSupport.LayoutCompleteEvent) {
         this.labelGroup.translationX = opts.series.rect.x;
+
+        this.updateHighlight();
     }
 
     private updateHighlight() {
         if (!this.enabled) return;
 
         const activeHighlight = this.highlightManager.getActiveHighlight();
+        const datum = activeHighlight?.datum ?? this.data?.at(-1);
 
-        if (activeHighlight == null) {
+        if (datum == null) {
             this.labelGroup.visible = false;
             return;
         }
 
         this.labelGroup.visible = true;
 
-        const datum = activeHighlight.datum;
+        const itemId = activeHighlight?.itemId;
```

Comment:
Just double checking, should this be `datum?.itemId`?

Reply:
https://github.com/ag-grid/ag-charts/pull/2372/files#diff-f8840d9e039fb128d737560b5bcbbe56bfeda18d24bea54bb311c178088410bdL406

It's refactoring this line